moyo-header

wxfharuka opened the pull request:
- [DEMO LINK](https://wxfharuka.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://wxfharuka.github.io/layout_moyo-header/report/html_report/)

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -15,8 +15,93 @@
+        alt="logo"
+      />
+      <nav class="nav">
+        <ul class="list">
+          <li class="list-item">
+            <a
+              href="index.html"
+              class="list-link"
```
Fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="index.html"
              class="nav__link"
```

volodymyr-soltys97 replied:
Also add this class for the first link 
<img width="809" alt="image" src="https://github.com/user-attachments/assets/f6c44518-c391-4f5a-9d13-f333c56378fe">

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -15,8 +15,93 @@
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="list-item">
```
Add the blank line between the links
```suggestion
          </li>
          
          <li class="list-item">
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```
```suggestion
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+  display: flex;
+  font-family: Roboto, sans-serif;
+  text-align: center;
+  margin: 0 auto;
+  max-width: 1024px;
+  width: 100%;
+  height: 60px;
+
+  /* box-sizing: border-box; */
+
+  padding: 0 20px;
```
```suggestion
  width: 100%;
  height: 60px;
  padding: 0 50px;
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+  padding: 0 20px;
+}
+
+img {
+  margin-right: 20%;
+  margin-left: 50px;
+}
```
```suggestion
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+  list-style: none;
+}
+
+li {
+  opacity: 100;
+  color: white;
+}
```
```suggestion
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+  position: relative;
+}
+
+.list-link[data-qa='hover']::before {
+  content: '';
+  position: absolute;
+  bottom: -5px;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 0;
+  height: 2px;
+  background-color: #00acdc;
+  transition: width 0.3s;
+}
```
```suggestion
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+  padding-bottom: 18.5px;
+}
+
+.list-item:first-child .list-link::after {
```
```suggestion
.is-active::after {
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+  background-color: #00acdc;
 }
+
+/* .list-item:first-child .list-link{
+  color:  #00acdc;
+} */
```
```suggestion
```

volodymyr-soltys97 commented on src/style.css:
```diff
@@ -1,3 +1,111 @@
+}
+
+.list-link:hover {
+  color: #00acdc;
```
Create a variable for repeated color and use it everywhere

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your code
You need to fix all failed tests. Fix the border position here 
<img width="386" alt="image" src="https://github.com/user-attachments/assets/dee9a653-b557-462d-a2e1-330863a21c10">

Remove the margin right for the last link 
<img width="187" alt="image" src="https://github.com/user-attachments/assets/bd9128c0-3fff-4f9f-93a5-f6e8bb92ded1">

The height menu must be 60px 
<img width="771" alt="image" src="https://github.com/user-attachments/assets/2248f935-a92b-4a74-bb3d-cd7ce2fd6ac0">

anastasiia-tilikina reviewed (changes requested):
the tests are still failed
Pls try to fix them
If you have any problems - feel free to ask help in chat

wxfharuka commented:
I have fixed the above errors and changed the code style, but there are still some problems with the tests, although my web page corresponds to the given layout in figma. please help me to fix the wrong aspects of my code so that the site works correctly

etojeDenys reviewed (changes requested):
<img width="1492" alt="image" src="https://github.com/user-attachments/assets/b4554393-6ad9-45d0-bc33-3a6210544c3b">
feel free to ask for some help in the platform chat, don't hesitate to do that

anastasiiavorobiova commented on src/index.html:
```diff
@@ -15,8 +15,104 @@
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <ul class="header-item">
+        <li class="header-link">
```
```suggestion

```

anastasiiavorobiova commented on src/index.html:
```diff
@@ -15,8 +15,104 @@
+              alt="logo"
+            />
+          </a>
+        </li>
+      </ul>
```
```suggestion

```

anastasiiavorobiova commented on src/index.html:
```diff
@@ -15,8 +15,104 @@
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
```
```suggestion
              class="nav__link"
```

anastasiiavorobiova commented on src/index.html:
```diff
@@ -15,8 +15,104 @@
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
```
```suggestion
              class="nav__link"
```

anastasiiavorobiova commented on src/index.html:
```diff
@@ -15,8 +15,104 @@
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
```
```suggestion
              class="nav__link"
```

anastasiiavorobiova commented on src/index.html:
```diff
@@ -15,8 +15,104 @@
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
```
```suggestion
              class="nav__link"
```

anastasiiavorobiova commented on src/index.html:
```diff
@@ -15,8 +15,104 @@
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link is-active"
```
```suggestion
              class="nav__link"
```

anastasiiavorobiova commented on src/style.css:
```diff
@@ -1,3 +1,105 @@
+  left: 50%;
+  transform: translateX(-50%);
+  border-radius: 8px;
+  width: 37px;
```
```suggestion
  width: 100%;
```

anastasiiavorobiova commented on src/style.css:
```diff
@@ -1,3 +1,105 @@
+  border-radius: 8px;
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
```
Consider using variables for repeating colors

anastasiiavorobiova commented on src/style.css:
```diff
@@ -1,3 +1,105 @@
+  padding-bottom: 18.5px;
+}
+
+a.is-active:first-child::after {
```
```suggestion
.is-active::after {
```

anastasiiavorobiova commented on src/style.css:
```diff
@@ -1,3 +1,105 @@
+  color: #00acdc;
+}
+
+.nav__item:first-child .nav__link {
```
```suggestion
.is-active {
```

anastasiiavorobiova commented on src/style.css:
```diff
@@ -1,3 +1,105 @@
+  color: #00acdc;
+}
+
+.nav__link[data-qa='hover'] {
```
```suggestion
.nav__link:hover {
```

anastasiiavorobiova reviewed (changes requested):
Great start! Added a few comments to fix
